From c231c591b3b86726d13786562b99969aed79f06b Mon Sep 17 00:00:00 2001
From: Arockiasathish S <arocs@qti.qualcomm.com>
Date: Tue, 18 Aug 2026 09:48:33 +0530
Subject: [PATCH] [QMESH] suppress compilation break due to BIO_flush
 unused-value warning

BIO_flush expands to (int)BIO_ctrl(...); discarding the result triggers
-Werror=unused-value. Cast to void to suppress the build error.

Signed-off-by: Arockiasathish S <arocs@qti.qualcomm.com>
---
 agent/src/beerocks/slave/son_slave_thread.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/src/beerocks/slave/son_slave_thread.cpp b/agent/src/beerocks/slave/son_slave_thread.cpp
index 79a1d4e23..4b624a39e 100644
--- a/agent/src/beerocks/slave/son_slave_thread.cpp
+++ b/agent/src/beerocks/slave/son_slave_thread.cpp
@@ -1070,7 +1070,7 @@ bool slave_thread::load_dpp_provisioned_keys()
         b64       = BIO_push(b64, bmem);
         BIO_set_flags(b64, BIO_FLAGS_BASE64_NO_NL);
         BIO_write(b64, spki_der_ptr, spki_len);
-        BIO_flush(b64);
+        (void)BIO_flush(b64);
         BUF_MEM *bptr = nullptr;
         BIO_get_mem_ptr(bmem, &bptr);
         std::string key_b64(bptr->data, bptr->length);
-- 
2.34.1

